src/EventStore.Core/ClusterVNode.cs

```diff
@@ -472,26 +472,38 @@ public class ClusterVNode :
 			});
 
 
-			List<IHttpAuthenticationProvider> httpAuthenticationProviders;
+			var httpAuthenticationProviders = new List<IHttpAuthenticationProvider>();
+
+			foreach (var authenticationScheme in _authenticationProvider.GetSupportedAuthenticationSchemes() ?? Enumerable.Empty<string>()) {
+				switch (authenticationScheme)
+				{
+					case "Basic":
+						httpAuthenticationProviders.Add(new BasicHttpAuthenticationProvider(_authenticationProvider));
+						break;
+					case "Bearer":
+						httpAuthenticationProviders.Add(new BearerHttpAuthenticationProvider(_authenticationProvider));
+						break;
+					case "Insecure":
+						httpAuthenticationProviders.Add(new PassthroughHttpAuthenticationProvider(_authenticationProvider));
+						break;
+					default:
+						Log.Warning($"Unsupported Authentication Scheme: {authenticationScheme}");
```

Comment:
How would Event Store react if none of the authentication schemes were supported? Is that something we want to check after this loop?

Reply:
@hayley-jean we could log an error in the case where none of the auth schemes are supported, what do you think?

cc @jageall

Reply:
Would this be a case where we would want to log an error and shut the node down?

Reply:
No supported auth scheme should result in the server shutting down

Reply:
@hayley-jean @jageall Fixed it in this commit: 0ebef5cf9c1515e918ac8151db8bb931b30ce777, thanks !